src/app.js

```diff
@@ -85,11 +85,9 @@ async function persistDumpData(sinkMeta) {
     const { clientId } = sinkMeta;
     let uniqueClientId = clientId;
 
-    if (saveEntryAssureUnique) {
```

Comment:
from what I recall the flag was here for cases where dynamo is not configured (such as running in debug mode), so we don't send if we don't have an instance. A more elegant solution would have been nice, but for now I think we need some sort of guard there.

Reply:
nvm, just saw the added check to saveEntryAssureUnique.